apps/mobile/.env

```
@@ -0,0 +1,6 @@
+BETTER_AUTH_SECRET=zRErXDDPlPrDfaljEPFVzXmCCyhBsFse
+BETTER_AUTH_URL=http://localhost:8081
+
+EXPO_PUBLIC_CONVEX_URL=https://good-puffin-21.convex.cloud
+EXPO_PUBLIC_SITE_URL=https://good-puffin-21.convex.site
+EXPO_PUBLIC_MOBILE_URL=exp://172.20.10.2:8081
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_MOBILE_URL key should go before the EXPO_PUBLIC_SITE_URL key

(UnorderedKey)